source/Files.cpp

```diff
@@ -575,6 +575,11 @@ void Files::OpenUserPluginFolder()
 	OpenFolder(Config() + "plugins");
 }
 
+// Open this user's save file directory in their native file explorer.
```

Comment:
If the style script hasn't complained yet, it will. There should be three spaces between each function